.graphqlrc.ts

```diff
@@ -44,7 +44,7 @@ module.exports = {
               FloatType: 'number',
               IntType: 'number',
               ItemId: 'string',
-              JsonField: 'unkown',
+              JsonField: 'unknown',
```

Comment:
This seems to be a typo which has been in the codebase for the last 15 months. Correcting this caused the EmbedBlock type to change (since it has a json field) so I also needed to fix that.

See changes in `src/blocks/EmbedBlock/EmbedBlock.astro`